tests/functional/c/cellvar_escaping_loop.py

```diff
@@ -123,6 +154,63 @@ def bad_case6():
     return lst
 
 
+def bad_case7():
+    """Multiple variables unpacked in comprehension."""
+    return [
+        lambda: (
+            x  # [cell-var-from-loop]
+            + y)  # [cell-var-from-loop]
```

Comment:
```suggestion
            x+ y)  # [cell-var-from-loop, cell-var-from-loop]
```
I don't know if it's intentional or if you want to differentiate x and y to see that a message is send for each, but you could also do that.

Reply:
@Pierre-Sassoulas I actually wasn't aware you could test for multiple error occurrences in the same line, good to know! 😅 

Since you've merged in this PR I'm assuming you don't actually need me to make this change, but I'll keep it in mind for the future.